core/server/apps/amp/lib/router.js

```diff
@@ -63,10 +63,11 @@ function getPostData(req, res, next) {
         }));
     }
 
-    helpers.entryLookup(urlWithoutSubdirectoryWithoutAmp, {permalinks}, res.locals)
+    // @NOTE: amp is not supported for static pages
```

Comment:
Why so? Do we plan to support amp for static pages?

Reply:
There is no reason, it's just not supported :) I don't think we will support it in the near future. We currently don't put any effort into the AMP feature, it's also off by default. There are some general community concerns of the AMP project ([e.g. see](https://github.com/ampproject/amphtml/issues/13597))